### ISSUE TITLE
Promos - General feedback

### ISSUE THREAD

**Issue by greenc05** [bbc/gel#40]
Some feedback after reading Promos https://bbc.github.io/code-gel/components/promos/

- The 2nd promo in the code example does not have alt text as stated: “Images can be considered decorative (alt=""; the first Promo in our example) or non-decorative (alt="[description of image]"; the second Promo in our example).”

- There is a typo for ‘direction’ the css in the tutorial ‘flex-diection: column;’

- Does this make sense? “This is possible by making the promo a Flexbox context”

- ~~Under ‘Recommended behaviour’ there is no explanation as to why I cannot have the whole card as the link area which is a very common pattern, for example using faux-block link (CSS solution) as per promo’s on the News homepage~~

- It might also be nice to talk about the use of the description list mark up and the visually hidden text that is used within that element, as dl's aren't commonly used and some people might not be aware of visually hidden text/aria-hidden... Also why is there no use of the time element for the timestamp?

- Would it also be worth while mentioning to remember to test your implementation with a range of assistive technology in the 'Important' section under 'Example implementation'?

**Comment by micmath** [comment]
The following issues  no longer apply (text has already been changed):

> The 2nd promo in the code example does not have alt text as stated: “Images can be considered decorative (alt=""; the first Promo in our example) or non-decorative (alt="[description of image]"; the second Promo in our example).”

> There is a typo for ‘direction’ the css in the tutorial ‘flex-diection: column;’

> Does this make sense? “This is possible by making the promo a Flexbox context”

**Comment by micmath** [comment]
I've added a footnote for
>It might also be nice to talk about the use of the description list mark up and the visually hidden text that is used within that element, as dl's aren't commonly used and some people might not be aware of visually hidden text/aria-hidden... Also why is there no use of the time element for the timestamp?

**Comment by micmath** [comment]
> Would it also be worth while mentioning to remember to test your implementation with a range of assistive technology in the 'Important' section under 'Example implementation'?

This seems like general advice that could conceivably apply to every piece of documentation we produce. It isn't related specifically to Promos, versus say Carousels. rather than adding such a note to every page, I think we should implement this as part of the `bbc/gel-check` project, which aims to present the GEL technical documentation in terms of testing.
